Sources/ClientRuntime/ClockSkew/ClockSkewStore.swift

```diff
@@ -0,0 +1,30 @@
+//
+// Copyright Amazon.com Inc. or its affiliates.
+// All Rights Reserved.
+//
+// SPDX-License-Identifier: Apache-2.0
+//
+
+import struct Foundation.TimeInterval
```

Comment:
This type is just a concurrency-safe store for clock skew values, keyed by the hostname of the server where that clock skew was measured.